Jenkinsfile_CNP

```diff
@@ -227,7 +227,13 @@ withPipeline(type, product, component) {
     env.TASK_MONITOR_API_URL = "https://wa-task-monitor-sptribs-case-api-pr-${CHANGE_ID}.preview.platform.hmcts.net"
     env.TASK_MANAGEMENT_API_URL = "https://wa-task-management-api-sptribs-case-api-pr-${CHANGE_ID}.preview.platform.hmcts.net"
     env.ROLE_ASSIGNMENT_API_URL = "https://am-org-role-mapping-service-sptribs-case-api-pr-${CHANGE_ID}.preview.platform.hmcts.net"
-    env.WA_FEATURE_ENABLED = true
+
+    def githubApi = new GithubAPI(this) as Object
```

Comment:
If we define this globally it could be used in this block and the afterSuccess akschartsinstall block meaning we don't need to create the gitHubApi twice?